androidbrowserhelper/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.6.2
+
+* The TWA launcher will no longer wait for the animation to finish before launching the Custom
+Tabs Activity, which improves launch time. This change does not affect the user-visible launch
+animation. The PR for the change is
+[#520](https://github.com/GoogleChrome/android-browser-helper/pull/520).
```

Comment:
nit: put the PR at the beginning of the line, as in other changelog entries. no "The PR for the change is" is needed.

Reply:
Done.